sdks/urbackend-python/test_sdk.py

```
@@ -0,0 +1,371 @@
+    no_auth_client = UrBackendClient(api_key=API_KEY, base_url=BASE_URL)
+    try:
+        no_auth_client.db.insert(
+            COLLECTION,
+            {"title": "Should Fail", "content": "No auth token"},
+            # deliberately NOT passing token
+        )
+        print("❌ Insert succeeded without auth — expected 401/403!")
+    except AuthError as e:
+        print(f"✅ Correctly blocked (AuthError): {e.message}")
+        print(f"   Status code: {e.status_code}")
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)